FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -518,7 +518,7 @@ - (void)evaluateStreamResponse:(NSDictionary *)response error:(NSError *)dataErr
       [self propogateErrors:error];
     } else {
       NSInteger clientTemplateVersion = [_configFetch.templateVersionNumber integerValue];
-      if (updateTemplateVersion >= clientTemplateVersion) {
+      if (updateTemplateVersion > clientTemplateVersion) {
```

Comment:
just to confirm - we have the same behavior on Android?

Reply:
Yeah it's like this already on Android, https://github.com/firebase/firebase-android-sdk/blob/cd8e0af2e8f462d7b3c88200b637580b18771ec8/firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigAutoFetch.java#L160